roles/ceph-validate/tasks/check_devices.yml

```diff
@@ -20,19 +20,20 @@
   register: dedicated_device_parted
   with_items: "{{ dedicated_devices }}"
   when:
-    - osd_scenario == 'non-collocated'
+    - dedicated_devices|default([]) | length > 0
```

Comment:
Is ``dedicated_devices`` still needed?

Reply:
Can't we use it anymore with lvm?

Reply:
the lvm scenarios only use ``devices`` or ``lvm_volumes``. I'm not sure what we'd use it for right now.

Reply:
hum, so what if I want to use a raw block device for wal/db, is that doable?

Reply:
batch decides where to best put block.db depending on the devices given. At this point batch doesn't do anything with wal I don't believe.